components/pages/farms/stake/index.tsx

```diff
@@ -65,30 +60,27 @@ export function Stake({ farm }) {
             }
             // console.log('approved:', current.pid, formatapprove)
 
-
             // LP Staked 
-            let lpStaked = await chadMasterContract.userInfo(current.pid, account).then(amount => {
```

Comment:
Since we are **awaiting** the promise, there's no need to use the callback with **then**. Removed the then callback and just moved the code out of it.